cmd/geth/usage.go

```diff
@@ -75,6 +75,7 @@ var AppHelpFlagGroups = []flagGroup{
 			utils.TestnetFlag,
 			utils.RinkebyFlag,
 			utils.SyncModeFlag,
+			utils.ExitWhenSyncedFlag,
 			utils.GCModeFlag,
```

Comment:
Please also add this flag to https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/main.go#L54

Reply:
This flag is there, may have been in wrong spot so moved it

Reply:
I mean you have to add this flag to https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/main.go#L54, otherwise, we can not parse the arg for it from the command line.

Reply:
And also, please add the console package back :)

Reply:
To clarify you mean add utils.ExitWhenSyncedFlag, to main.go?  I believe it is there already at line 88?

Reply:
Oh yes it is. My mistake.